deploy/charts/nfspv-provisioner/templates/NOTES.txt

```diff
@@ -0,0 +1,12 @@
+Thank you for installing {{ .Chart.Name }} 😀
+
+Your release is named {{ .Release.Name }} and it's installed to namespace: {{ .Release.Namespace }}.
+
+The OpenEBS NFSPV Provisioner has been installed check its status by running:
+$ kubectl get pods -n {{ .Release.Namespace }}
+
+Use `kubectl get bd -n {{ .Release.Namespace }} ` to see the list of
+blockdevices attached to the Kubernetes cluster nodes.
+
+For more information, visit our Slack at https://openebs.io/community or view
+the documentation online at http://docs.openebs.io/.
```

Comment:
NFS provisioner docs are not available at docs.openebs.io at the moment. Could we use https://github.com/openebs/dynamic-nfs-provisioner/blob/develop/README.md instead?

Reply:
Pointed to the developer readme file as you suggested , Thanks

Reply:
Thanks mate , Updated to the developer readme file as you suggested